.github/workflows/build-linux.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: fragcolor-xyz/shards
-          fetch-depth: 1
+          fetch-depth: 2
```

Comment:
Is that necessary? As far as I know, it was only required when doing coverage so that the codecov action can calculate the commit hash.

Reply:
we are doing coverage here

Reply:
Not on that job, no. Coverage is done on the next job.

Reply:
Yeah wrong file, wanted to add it in the gpu test file